src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx

```diff
@@ -1,15 +1,20 @@
 import { LoadingOutlined } from "@ant-design/icons";
-import { Button } from "antd";
+import { Button, Checkbox } from "antd";
+import { CheckboxChangeEvent } from "antd/es/checkbox";
 import * as React from "react";
 import { useDispatch, useSelector } from "react-redux";
 
 import { closeUpload } from "../../../state/route/actions";
 import { getSelectedUploads } from "../../../state/selection/selectors";
+import { getShouldStoreLocally } from "../../../state/setting/selectors";
```

Comment:
`shouldBeInLocal` is not the greatest name choice, but I think it makes sense to be consistent and use that name rather than adding a new name (`getShouldStoreLocally`) which represents the same info

Reply:
I dont know if I agree with this. The state that we are defining is that new uploads should store locally meaning that it should pass true to new calls that involve shouldBeInLocal, but isnt the exact flag itsself.

Reply:
The distinction I was making was:

shouldStoreLocally = store all new uploads made by app locally 

versus the specific tag

should_be_in_local  which is the metadata value

I honestly dont have too strong of an opinion and am happy to change it. just like this message if I should!

Reply:
> store all new uploads made by app locally

Sorry this doesnt clear it up for me.  

Doesn't 'store new upload locally' mean the same thing as 'set `should_be_in_local` equal to true for the upload' ?  

Im not trying to be difficult, I really am trying to understand the distinction.

If they mean something different, I can understand a different variable name, although it will be confusing to have 2 variables with such similar names.  If it really is distinct, I would suggest making the name less similar to the existing variable name.

Reply:
The distinction I think Brian is trying to make is that this flag is like a setting rather than a direct selection of the toggle. I think either are fine for this since it isn't common for server variable names to directly map to client state variable names in the app.